src/App.css

```diff
@@ -26,3 +26,82 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+input[type=text], input[type=tel], input[type=email], select {
+    width: 100%;
+    padding: 12px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+
+input[type=submit],input[type=button] {
```

Comment:
Make sure to split multiple style definitions in to different lines

Reply:
ok